lib/routes/bankless/article.ts

```diff
@@ -0,0 +1,56 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+const baseUrl = 'https://www.bankless.com/read';
+export const route: Route = {
+    path: '/article',
+    categories: ['finance'],
+    example: '/bankless/article',
+    radar: [
+        {
+            source: ['bankless.com/read'],
+        },
+    ],
+    name: 'Bankless articles',
+    maintainers: ['Colin'],
+    handler,
+};
+async function handler() {
+    const response = await ofetch(baseUrl);
+    const $ = load(response);
+
+    const list = $('.item.articleBlockSmall')
+        .toArray()
+        .slice(0, 40)
+        .map((u) => {
+            const $u = $(u);
+            const item = {
+                title: $u.find('.title').text(),
+                link: $u.attr('href'),
+            };
+            return item;
+        });
+    const items = await Promise.all(
+        list.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const response = await ofetch(item.link);
+                const $ = load(response);
+                const urlList = $('#article').first();
+                const $u = $(urlList);
+                item.description = $u.find('#focalAnchor.contents').text();
```

Comment:
Please use `.html()` instead. It is also a good idea to include the image `.postImgWrapper` as well

Reply:
ok, fixed.